src/services/database/registration-request-repository.js

```diff
@@ -2,13 +2,23 @@ import ModelFactory from '../../models';
 import { modelName } from '../../models/registration-request';
 import { runWithinTransaction } from './helper';
 import { logInfo } from '../../middleware/logging';
+import {NhsNumberNotFoundError} from "../../errors/errors";
```

Comment:
Incredibly small request for consistency! Totally optional 😄

```suggestion
import { NhsNumberNotFoundError } from "../../errors/errors";
```